test-setup.js

```diff
@@ -6,76 +6,96 @@
 // import { configure } from 'enzyme';
 // import dateMock from './mocks/dateMock';
 
+import i18n from 'i18next';
+import reactI18n from 'react-i18next';
+
 /**
  * lets mock i18next for all tests
  */
-jest.mock('i18next', () => {
-	const noop = () => {};
-	const i18n = {
-		t: (msg, options) => {
-			let buff = options.defaultValue || msg;
-			const split = buff.split('}}');
-			if (split.length > 1) {
-				buff = split.reduce((acc, current) => {
-					const sub = current.split('{{');
-					let value = sub.length > 1 ? options[sub[1].trim()] : '';
-					if (value === undefined) {
-						value = '';
-					}
-					return `${acc}${sub[0]}${value}`;
-				}, '');
-			}
-			return buff;
-		},
-		isMock: true,
-		getFixedT: () => i18n.t,
-		options: {},
-		language: 'en',
-		languages: ['en'],
-		isInitialized: true,
-		on: noop,
-		off: noop,
-		loadNamespaces: noop,
-		hasResourceBundle: () => false,
-		services: {
-			resourceStore: {
-				data: {},
-			},
-			backendConnector: {},
-		},
-		store: {
-			data: {},
-			on: noop,
-			off: noop,
-		},
-		changeLanguage: noop,
-	};
-	i18n.init = () => {};
-	return {
-		...i18n,
-		createInstance: () => i18n,
-	};
-});
-
-// function getMajorVersion() {
-// 	if (!process.env.REACT_VERSION) {
-// 		return '16';
-// 	}
-// 	return process.env.REACT_VERSION.replace('^', '').split('.')[0];
-// }
-
-// const REACT_VERSION = getMajorVersion();
+// jest.mock('i18next', () => {
```

Comment:
you keep the comments ?

Reply:
For now yes. This file is loaded by all the packages in jest configuration, and it will surely break a lot of tests. So I keep the old code for now to easily access to it and pick what I need when I go over the packages. The clean will happen at the end